### PR TITLE
Golang.yaml: Adds new language definition

### PR DESCRIPTION
**Pull request by SerekKiri**
Golang.yaml --> Adds new language definition named Golang

Source:
[coala/golang.py](https://github.com/coala/coala/blob/master/coalib/bearlib/languages/definitions/Golang.py)
[WikiData](https://www.wikidata.org/wiki/Wikidata:Main_Page)

### CODE REVIEW

File: data/Language/Golang.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - interface
+  - map
+  - package
+  - range
+  - return
+  - select
+  - struct
+  - switch
+  - type
+  - var
```
Comment (shikharvaish28) [inline data/Language/Golang.yaml]: Add a blank line here.

**Comment by shikharvaish28** [comment]
Please rebase you branch. @SerekKiri

File: data/Language/Golang.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+identifier: Golang
+wikidata: Q37227
+aliases:
+  - go
+extensions:
+  - .go
```
Comment (abhishalya) [inline data/Language/Golang.yaml]: Remove the `.`. It is of no use.

**Comment by abhishalya** [comment]
@SerekKiri Also, always keep your master always clean. Instead create branch for the stuff you are changing.

**Comment by SerekKiri** [comment]
@abhishalya I know, but I just forgot 😄 . And I can't understand why this failed when I removed one dot. Before it passed.

**Comment by SerekKiri** [comment]
@abhishalya anything more need changes?

**Comment by yukiisbored** [comment]
It's officially "Go" not "Golang".
Just use "golang" as one of it's aliases.

Other than that, looks good to me. Not sure what's up with Statik.

**Comment by yukiisbored** [comment]
Checked the keywords with the language spec. Seems like everything is in there.
https://golang.org/ref/spec

**Comment by yukiisbored** [comment]
ack https://github.com/coala/coAST/commit/55e0d9dbaf32c24a9bf94aac6c412a640a6cf733

**Comment by SerekKiri** [comment]
Updated to Go 👍

**Comment by yukiisbored** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by yukiisbored** [comment]
Why is John gitmate again...

**Pull request merged**